### PR TITLE
#487 :set passthrough_enabled = true by default 

### PR DESCRIPTION
**Pull request by anandrgitnirman**
by default enforce this behavoir

set passthrough_enabled = true

the user also needs to specify a valid URL for the config passthrough_endpoint

user has the choice to set the value to false explicitly

### CODE REVIEW

**Comment by arturgontijo** [comment]
Cool!
My idea was is one sets `passthrough_endpoint` with a valid URL, Daemon should automatically sets `passthrough_enabled` to true.
If I got your update right, it's doing it. =)

File: config/config.go [hunk 1]
```diff
@@ -307,9 +307,10 @@ func ValidateEmail(email string) bool {
 }
 
 func ValidateEndpoints(daemonEndpoint string, passthroughEndpoint string) error {
+
 	passthroughURL, err := url.Parse(passthroughEndpoint)
-	if err != nil {
-		return errors.New("passthrough endpoint must be a valid URL")
+	if err != nil || passthroughURL.Host == "" {
```
Comment (raamb) [inline config/config.go:312]: Lets add the isValidURL check as well?

File: config/config.go [hunk 2]
```diff
@@ -307,9 +307,10 @@ func ValidateEmail(email string) bool {
...
+		return errors.New("passthrough_endpoint must be a valid URL")
```
Comment (raamb) [inline config/config.go:313]: Lets make the error message descriptive. Something like- You need to set the passthrough_endpoint variable in the daemon config to the endpoint of your AI service"

File: config/config.go (hunk 1 above)
Comment (anandrgitnirman) [reply config/config.go:312]: passthroughURL, err := url.Parse(passthroughEndpoint) , this line takes care of it  
isValidURL will just return a boolean , 
and we also need to do additiona checks in the function from the passthroughURL struct 
For example 
the port number after parsing to make sure the daemon port and the service port are not the same

File: config/config.go (hunk 2 above)
Comment (anandrgitnirman) [reply config/config.go]: done

**Pull request merged**